src/main/frontend/state.cljs

```diff
@@ -941,6 +941,10 @@
 (defn set-theme!
   [theme]
   (set-state! :ui/theme theme)
+  (when (mobile-util/native-ios?)
```

Comment:
Does android not support this?

Reply:
It should work on most Android modals (tested on Emulator), though never work on my Mi 9. 
However, a proper background color should also be hard-coded for better display on Android, thus I only added this feature on iOS.